src/compas/data/data.py

```diff
@@ -59,16 +121,17 @@ def __setstate__(self, state):
 
     @property
     def DATASCHEMA(self):
```

Comment:
These properties are not present in the doc output. At least locally, did I miss some config? I have latest versions of both sphinx and our sphinx theme.

![image](https://user-images.githubusercontent.com/933277/148189725-92eb003a-a065-4f06-8084-c7f8abc80dc9.png)

Reply:
no indeed. figured they are to much of an experimental developer thing to expose in the docs for now...